modules/ROOT/nav.adoc

```diff
@@ -24,138 +24,12 @@ include::guides:partial$nav.adoc[]
-    ***** xref:n1ql:n1ql-language-reference/indexing-meta-info.adoc[]
-    ***** xref:n1ql:n1ql-language-reference/index-partitioning.adoc[]
-   **** xref:n1ql:n1ql-language-reference/createprimaryindex.adoc[]
-   **** xref:n1ql:n1ql-language-reference/createscope.adoc[]
-   **** xref:n1ql:n1ql-language-reference/delete.adoc[]
-   **** xref:n1ql:n1ql-language-reference/dropcollection.adoc[]
-   **** xref:n1ql:n1ql-language-reference/dropfunction.adoc[]
-   **** xref:n1ql:n1ql-language-reference/dropindex.adoc[]
-   **** xref:n1ql:n1ql-language-reference/dropprimaryindex.adoc[]
-   **** xref:n1ql:n1ql-language-reference/dropscope.adoc[]
-   **** xref:n1ql:n1ql-language-reference/execute.adoc[]
-   **** xref:n1ql:n1ql-language-reference/execfunction.adoc[]
-   **** xref:n1ql:n1ql-language-reference/explain.adoc[]
-   **** xref:n1ql:n1ql-language-reference/grant.adoc[]
-   **** xref:n1ql:n1ql-language-reference/infer.adoc[]
-   **** xref:n1ql:n1ql-language-reference/insert.adoc[]
-   **** xref:n1ql:n1ql-language-reference/merge.adoc[]
-   **** xref:n1ql:n1ql-language-reference/prepare.adoc[]
-   **** xref:n1ql:n1ql-language-reference/revoke.adoc[]
-   **** xref:n1ql:n1ql-language-reference/rollback-transaction.adoc[]
-   **** xref:n1ql:n1ql-language-reference/savepoint.adoc[]
-   **** xref:n1ql:n1ql-language-reference/selectintro.adoc[SELECT]
-    ***** xref:n1ql:n1ql-language-reference/select-syntax.adoc[]
-    ***** xref:n1ql:n1ql-language-reference/selectclause.adoc[]
-    ***** xref:n1ql:n1ql-language-reference/with.adoc[]
-    ***** xref:n1ql:n1ql-language-reference/from.adoc[]
-    ***** xref:n1ql:n1ql-language-reference/hints.adoc[]
-    ***** xref:n1ql:n1ql-language-reference/join.adoc[]
-    ***** xref:n1ql:n1ql-language-reference/nest.adoc[]
-    ***** xref:n1ql:n1ql-language-reference/unnest.adoc[]
-    ***** xref:n1ql:n1ql-language-reference/comma.adoc[]
-    ***** xref:n1ql:n1ql-language-reference/let.adoc[]
-    ***** xref:n1ql:n1ql-language-reference/where.adoc[]
-    ***** xref:n1ql:n1ql-language-reference/groupby.adoc[]
-    ***** xref:n1ql:n1ql-language-reference/window.adoc[]
-    ***** xref:n1ql:n1ql-language-reference/union.adoc[]
-    ***** xref:n1ql:n1ql-language-reference/orderby.adoc[]
-    ***** xref:n1ql:n1ql-language-reference/limit.adoc[]
-    ***** xref:n1ql:n1ql-language-reference/offset.adoc[]
-   **** xref:n1ql:n1ql-language-reference/set-transaction.adoc[]
-   **** xref:n1ql:n1ql-language-reference/update.adoc[]
-   **** xref:n1ql:n1ql-language-reference/updatestatistics.adoc[]
-    ***** xref:n1ql:n1ql-language-reference/statistics-expressions.adoc[]
-    ***** xref:n1ql:n1ql-language-reference/statistics-index.adoc[]
-    ***** xref:n1ql:n1ql-language-reference/statistics-indexes.adoc[]
-    ***** xref:n1ql:n1ql-language-reference/statistics-delete.adoc[]
-   **** xref:n1ql:n1ql-language-reference/upsert.adoc[]
-
 
+include::n1ql:partial$nav.adoc[]
 +
 --
 include::javascript-udfs:partial$nav.adoc[]
 --
```

Comment:
I would get rid of this include and just incorporate the `javascript-udfs` nav into the `n1ql` nav, in exactly the same way as you did with the `learn` partial above. Makes things a lot simpler.

```suggestion

```